src/govuk/template.test.js

```diff
@@ -152,6 +153,18 @@ describe('Template', () => {
       expect($('body > div:last-of-type').text()).toEqual('bodyEnd')
     })
 
+    describe('inline script that adds "js-enabled" class', () => {
+      it('should match the hash published in docs. A change to the script would be a breaking change, and it would also require updating the hash in https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#javascript', () => {
+        const $ = renderTemplate()
+        const script = $('body > script').first().html()
+
+        // Create a base64 encoded hash of the contents of the script tag
+        const hash = crypto.createHash('sha256').update(script).digest('base64')
+
+        expect('sha256-' + hash).toEqual('sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=')
```

Comment:
Just because this is so long, I wonder if we should consider splitting out into a comment directly above the hash? If someone edited the test, they'd still see the comment.

Thoughts?

```suggestion
      it('should match the hash published in docs', () => {
        const $ = renderTemplate()
        const script = $('body > script').first().html()

        // Create a base64 encoded hash of the contents of the script tag
        const hash = crypto.createHash('sha256').update(script).digest('base64')

        // A change to the script would be a breaking change, and it would also
        // require updating the hash in https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#javascript
        expect('sha256-' + hash).toEqual('sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=')
```

Reply:
@36degrees Good idea, I'll change it 👍